### PR TITLE
Update category summaries

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -2,55 +2,57 @@
...
 '''
 {{< /card >}}
 
 {{< card >}}
-title = 'Community & organization'
+title = 'For Organization'
 link = '/community'
 body = '''
-...
+Calender, forum, and other services for organization the community.
```
Comment (bsipocz) [inline content/_index.md:33]:
```suggestion
Calendar, forum, and other services for organizing the community.
```

File: content/_index.md [hunk 2]
```diff
@@ -2,55 +2,57 @@
...
 '''
 {{< /card >}}
 
 {{< card >}}
-title = 'Analytics'
+title = 'For Insight'
 link = '/analytics'
 body = '''
-...
+Through analytics, better understand project development and community health.
```
Comment (bsipocz) [inline content/_index.md:41]:
```suggestion
Through analytics, to better understand project development and community health.
```

File: content/_index.md [hunk 3]
```diff
@@ -2,55 +2,57 @@
...
 '''
 {{< /card >}}
 
 {{< card >}}
-title = 'GitHub Actions'
+title = 'For GitHub'
 link = '/actions'
 body = '''
-...
+Actions for automating various aspects of project development.
```
Comment (bsipocz) [inline content/_index.md:49]:
```suggestion
GitHub Actions for automating various aspects of project development.
```

**Review by bsipocz — APPROVED** [review]
minor nitpicks

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**